src/clj/test/cards/operations.clj

```diff
@@ -1358,3 +1358,19 @@
       (prompt-select :corp (refresh vanilla))
       (is (not= "Wetwork Refit" (:title (first (:hosted (refresh vanilla)))))
           "Wetwork Refit is not hosted on Vanilla"))))
+
+(deftest offer-you-cant-refuse
+  ;; An Offer You Can't Refuse - exact card added to score area, not the last discarded one
+  (do-game
+    (new-game (default-corp [(qty "Celebrity Gift" 1) (qty "An Offer You Can't Refuse" 1)])
+              (default-runner))
+    (play-from-hand state :corp "An Offer You Can't Refuse")
+    (prompt-choice :corp "R&D")
+    (core/move state :corp (find-card "Celebrity Gift" (:hand (get-corp))) :discard)
+    (is (= 2 (count (:discard (get-corp)))))
+    (prompt-choice :runner "No")
+    (is (= 1 (:agenda-point (get-corp))) "An Offer the Runner refused")
+    (is (= 1 (count (:scored (get-corp)))))
+    (is (= "An Offer You Can't Refuse" (:title (first (:scored (get-corp))))))
```

Comment:
I was originally trying to do something like that, but didn't know you could use 

`(is (find-card ..))` 

directly. I can have it changed to follow that approach instead.